### ISSUE TITLE
Unable to edit or create any model with a "Comments" field due to a type error

### ISSUE THREAD

**Issue by mjadczuk** [VirtusLab-Open-Source/strapi-plugin-comments#175]
I've installed the plugin according to the README instruction, enabled it for specific models, and added the custom "Comments" field for them in the content type builder via admin panel. When I'm trying to create a new model or edit an existing model with the "Comments" field, it displays a generic error page and the following error is printed in the console:

```
TypeError: Cannot read properties of null (reading 'commentsNumber')
    at assertCorrectState (main.2735eaac.js:84829:20)
    at fromInput (main.2735eaac.js:84819:3)
    at CustomFieldInput (main.2735eaac.js:84689:95)
// (...) obfuscated functions
```

The attribute schema is as follows:

```json
  "attributes": {
    "comments": {
      "type": "customField",
      "customField": "plugin::comments.comments"
    }
  }
```

Is there anything I might have missed in my configuration that would cause such an error?

**Comment by mjadczuk** [comment]
I can confirm that the plugin works with Strapi 4.4.7, but fails with the error that I've posted on Strapi 4.5.3.

**Comment by cyp3rius** [comment]
Investigation in progress

**Comment by cyp3rius** [comment]
@mjadczuk fixed in `v2.2.3`
